src/clj/rems/api/services/category.clj

```diff
@@ -14,7 +14,7 @@
   (category/get-categories))
 
 (defn- check-category-children [children]
-  (when-let [not-found (seq (remove (comp category/get-category :category/id) children))]
+  (when-let [not-found (seq (remove #(category/get-category (:category/id %)) children))]
```

Comment:
i think there are two points to this:
1) explicit fn is more linear to reason about, no reverse order to think about. we have talked about using "simple" syntactical constructs and while comp is often tempting, it might be off-putting if we mostly use threading macros or anonymous functions
2) performance-wise i would think that anonymous fn is faster because comp needs to dispatch additional function calls. here the difference is likely unnoticeable though

Reply:
I've preferred some `comp` for "idiomatic" case like `(filter (comp #{val} :get-val) xs)`. I think both of the styles fit REMS codebase.